app/models/user.rb

```diff
@@ -1,10 +1,14 @@
 class User < ApplicationRecord
-  has_many :posts, foreign_key: 'user_id'
-  has_many :comments, foreign_key: 'user_id'
-  has_many :likes, foreign_key: 'user_id'
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] In order to have all the functionality work, some attributes need to be present on the devise list. ` :confirmable` is one of the attributes that will help with user email confirmation.